web/annotation_editor_layer_builder.js

```diff
@@ -52,7 +54,6 @@ class AnnotationEditorLayerBuilder {
    * @param {AnnotationEditorLayerBuilderOptions} options
    */
   constructor(options) {
-    this.pageDiv = options.pageDiv;
```

Comment:
Please *also* remove https://github.com/mozilla/pdf.js/blob/f891803e15524237b22d2c84b11ca39fadaaceda/web/annotation_editor_layer_builder.js#L129

Reply:
This still seems in addressed, when looking at the diff.